.github/workflows/fetch-observability-packs.yml

```diff
@@ -0,0 +1,100 @@
+name: Fetch Observability Packs
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: '0 0 * * *' # midnight
+
+env:
+  BOT_NAME: nr-opensource-bot
+  BOT_EMAIL: opensource+bot@newrelic.com
+  NODE_OPTIONS: '--max-old-space-size=4096'
+
+jobs:
+  fetch-observability-packs:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repo
+        uses: actions/checkout@v2
+        with:
+          ref: main
+
+      - name: Setup node.js
+        uses: actions/setup-node@v1
+        with:
+          node-version: 12
+
+      - name: Cache dependencies
+        id: yarn-cache
+        uses: actions/cache@v2
+        with:
+          path: '**/node_modules'
+          key: ${{ runner.os }}-node-modules-${{ hashFiles('**/yarn.lock') }}
+
+      - name: Install dependencies
+        if: steps.yarn-cache.outputs.cache-hit != 'true'
+        run: yarn install --frozen-lockfile
+
+      - name: Fetch observability packs
+        run: yarn run fetch-observability-packs
+        env:
+          NR_GQL_URL: 'https://api.newrelic.com/graphql'
```

Comment:
Super nitpicky, but would you mind renaming this to `NR_API_URL` to stay consistent with the other variable we're setting on the environment?

Reply:
Done!